docs/src/modules/utils/helpers.test.js

```diff
@@ -11,6 +11,7 @@ import InputLabel from '@material-ui/core/InputLabel';
 import FormControl from '@material-ui/core/FormControl';
 import FormHelperText from '@material-ui/core/FormHelperText';
 import Select from '@material-ui/core/Select';
+import SliderUnstyled from '@material-ui/unstyled/SliderUnstyled';
```

Comment:
Should this change be motived into a different pull request?

Reply:
Done